### PR TITLE
Add movie title to the card

### PR DESCRIPTION
**Pull request by JennyWeij**
[X] Added the movie title to the moviecard

### CODE REVIEW

File: src/Pages/HomePage.tsx [hunk 1]
```diff
@@ -1,12 +1,11 @@
 import { Box, Typography } from '@mui/material'
 import { useState } from 'react'
-import movieData from '../../data/movies.json'
+import { default as movieData } from '../../data/movies.json'
```
Comment (linusHernvall) [inline src/Pages/HomePage.tsx:3]: I can't reach your branch for some reason, but do we need to change the name on movieData?

Comment (JennyWeij) [reply src/Pages/HomePage.tsx:3]: I am not sure where this updated import of movieData is coming from... let's check with the other group members.

**Comment by nathaliegustafsson** [comment]
Looks nice! I just noticed that the cards have different heights depending on how much content they have, maybe we can add the same height on all of them?

Skärmavbild 2023-12-12 kl  11 19 27

**Comment by linusHernvall** [comment]
> Looks nice! I just noticed that the cards have different heights depending on how much content they have, maybe we can add the same height on all of them?
> 
> Skärmavbild 2023-12-12 kl 11 19 27

You can use the same height that we have used on the cards in the carousel, if you like - height: '100px'. 😃

**Review by nathaliegustafsson — APPROVED** [review]
Snyggt!

**Pull request merged**